sisl/supercell.py

```diff
@@ -925,6 +925,12 @@ def __str__(self):
         s = ',\n '.join(['ABC'[i] + '=[{:.3f}, {:.3f}, {:.3f}]'.format(*self.cell[i]) for i in (0, 1, 2)])
         return self.__class__.__name__ + ('{{nsc: [{:} {:} {:}],\n ' + s + ',\n}}').format(*self.nsc)
 
+    def __repr__(self):
+        r = lambda x: round(x, 3)  # rounding for display (shorter than fmt-string when numbers are already round)
+        a, b, c = map(r, np.linalg.norm(self.cell, axis=1))
+        alpha, beta, gamma = (r(self.angle(i, j)) for i, j in ((1, 2), (0, 2), (0, 1)))
```

Comment:
We have `SuperCell.parameters` for this ;)

I'll merge and change.

Reply:
@jonaslb also, you should try and avoid `np.linalg.norm`, I found that it is much slower than doing it manually `(xyz ** 2).sum(0) ** 0.5`, probably since there is tons of redundant code in `linalg.norm` (it has other good uses).